packages/core/package.json

```diff
@@ -45,5 +46,12 @@
     "test": "jest",
     "build": "tsup src/index.ts --format esm,cjs --dts",
     "dev": "tsup src/index.ts --format esm,cjs --dts --watch"
+  },
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
+    }
```

Comment:
@yisding got these from https://antfu.me/posts/publish-esm-and-cjs - i tested `clip.ts` with importing llamaindex as ESM and as CJS module - both worked